labelbox/data/serialization/ndjson/objects.py

```diff
@@ -582,7 +616,8 @@ def from_common(
         cls,
         annotation: Union[ObjectAnnotation, List[List[VideoObjectAnnotation]],
                           VideoMaskAnnotation], data: Union[ImageData, TextData]
-    ) -> Union[NDLine, NDPoint, NDPolygon, NDRectangle, NDMask, NDTextEntity]:
+    ) -> Union[NDLine, NDPoint, NDPolygon, NDDocumentRectangle, NDRectangle,
```

Comment:
why? not clear how a pdf rectangle can annotate video

Reply:
It can't annotate video, `NDDocumentRectangle` is only applicable for `ObjectAnnotation`

Reply:
was confused by PR lines